<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>854</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>99</y>
     <width>391</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="jenisBarangLabel">
      <property name="text">
       <string>ID Barang</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="editJenis"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="namaBarangLabel">
      <property name="text">
       <string>Nama Barang</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="editNamaBarang"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="satuanLabel">
      <property name="text">
       <string>Satuan</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="editSatuan"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnSimpan">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>220</y>
     <width>91</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>SIMPAN</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnUbah">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>220</y>
     <width>91</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>UBAH</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnHapus">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>220</y>
     <width>91</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>HAPUS</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tabelJenisBarang">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>260</y>
     <width>381</width>
     <height>192</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>ID BARANG</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Jenis Barang</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Satuan</string>
    </property>
   </column>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
